src/core/ComboBox/ComboBox.test.tsx

```diff
@@ -319,26 +339,28 @@ it('should work with custom itemRenderer', () => {
   });
   const input = assertBaseElement(container);
 
-  input.focus();
-  getByText('CUSTOM Item 1').click();
+  userEvent.tab();
+  userEvent.click(getByText('CUSTOM Item 1'));
   expect(mockOnChange).toHaveBeenCalledWith(1);
   expect(document.querySelector('.iui-menu')).not.toBeVisible();
   expect(input).toHaveValue('Item 1'); // the actual value of input doesn't change
 
-  input.blur();
-  input.focus();
+  userEvent.tab({ shift: true });
+  userEvent.tab();
+
   expect(
     document.querySelector(
       '.iui-menu-item.iui-active.iui-focused.my-custom-item',
     ),
   ).toHaveTextContent('CUSTOM Item 1');
 
-  fireEvent.keyDown(input, { key: 'ArrowDown' });
+  userEvent.keyboard('{ArrowDown}');
+
   expect(
     document.querySelector('.iui-menu-item.iui-focused.my-custom-item'),
   ).toHaveTextContent('CUSTOM Item 2');
```

Comment:
I don't understand what is up with these keyboard navigation tests, I can't get them to pass.